Interop/Runtime/src/native/kotlin/kotlinx/cinterop/Pinning.kt

```diff
@@ -47,6 +47,9 @@ inline fun <T : Any, R> T.usePinned(block: (Pinned<T>) -> R): R {
 fun Pinned<ByteArray>.addressOf(index: Int): CPointer<ByteVar> = this.get().addressOfElement(index)
 fun ByteArray.refTo(index: Int): CValuesRef<ByteVar> = this.usingPinned { addressOf(index) }
 
+fun Pinned<CharSequence>.addressOf(index: Int): CPointer<ByteVar> = this.get().addressOfElement(index)
```

Comment:
`String`, not `CharSequence`, seems there's no way to do that for generic char sequence, as we must know implementation details.